install/build.py

```diff
@@ -112,10 +113,12 @@ def check_cudnn_version(compiler, settings):
 
     cudnn_version = int(out)
 
-    if cudnn_version < minimum_cudnn_version:
+    if not minimum_cudnn_version <= cudnn_version <= maximum_cudnn_version:
+        min_major = minimum_cudnn_version // 1000
+        max_major = maximum_cudnn_version // 1000
         utils.print_warning(
-            'cuDNN version is too old: %d' % cudnn_version,
-            'cuDNN v2 or newer is required')
+            'Unsupported cuDNN version: %d' % cudnn_version,
+            'cuDNN v%d<= and <=v%d is required' % (min_major, max_major))
```

Comment:
I think it's better not to round the numbers.
We can't assume the API won't change in the middle of a major version series (can we?).
In that case, if rounded, we can't  present the precise supported versions to the user.
(Also `maximum_cudnn_version` should be an actual version)